src/sdk/model/vapp/vapp.ts

```diff
@@ -686,6 +689,32 @@ export class Vapp extends Entity implements EntityWithPerfSamples {
     });
   }
 
+  /**
+   * Get the Vapp's product sections
+   * @returns {Promise<Array<ProductSection>>} a promise that resolves with the product sections of the VM
```

Comment:
*product sections of Vapp.

Everything else looks good!

Reply:
Fixed @SuperConfuserUser